File: CP Lab/Lab4/3.c
# include <stdio.h>

int main(){
    
    float side1, side2, side3;

    
    printf("Enter the length of the first side:");
    scanf("%f", &side1);
    printf("Enter the length of the second side:");
    scanf("%f", &side2);
    printf("Enter the length of the third side:");
    scanf("%f", &side3);
    
    if ((side1+side2>side3) && (side1+side3>side2) && (side2+side3>side1)) {
        printf("This is a VALID Triangle");
    }
    
    else {
        printf("This is an INVALID Triangle");
    }
  
    return 0;
}

File: CP Lab/Lab3/3.py
# Lab Project by Sarang Dev Saha 

# LAB_03 QUE_03
#Write a program to input sides of a triangle and check whether triangle is valid or not

#Taking input from the user
side1=float(input("Enter the length of side 1:"))
side2=float(input("Enter the length of side 2:"))
side3=float(input("Enter the length of side 3:"))
#A triangle is valid as long as the sum of two sides is greater than the third
#Condition for the validity of the triangle
if (side1+side2>side3) and (side2+side3>side1) and (side3+side1>side2):
	print("The above mentioned sides are VALID for a triangle")
#Condition for the invalidity of the triangle
else:
	print("The above mentioned sides are INVALID for a triangle")